### DISCUSSION TITLE
Firma lezione docente

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#196] [Richieste di aiuto]
Buonasera ,quando parlo della firma da far apporre al docente prima di inserire la lezione del giorno, i voti e l'appello , intendo  una cosa molto semplice, riporto l'allegato per rendere idea,dal momento che in questo registro manca proprio l'atto di andare a firmare 
https://user-images.githubusercontent.com/93823125/156852267-3b31535e-53b1-40b0-899b-d48acce2c399.png

**Comment by manuelbarberi** [comment]
Come si chiama questo registro da dove hai estrapolato la foto?

**Comment by sempliciniluigi89** [comment]
Il registro si chiama Mastercom,ho estrapolato la foto da un video

**Comment by manuelbarberi** [comment]
Si
